jest.config.build.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+	moduleNameMapper: {
+		"src/.*": "<rootDir>/dist/immer.esm.js"
```

Comment:
Interesting. Shouldn't you refer to `src` in test files and let it translated to dist with this?

Reply:
yeah, this makes the tests behave as if immer was a dependency. So if anything is referred from src that is also exported from the index, it is fine, otherwise it will become undefined, so any graybox tests are marked to be not included in this build config